webapp/app.py

```diff
@@ -793,6 +794,9 @@ def takeovers_index():
 app.add_url_rule("/cube", view_func=cube_home)
 app.add_url_rule("/cube/microcerts", view_func=cube_microcerts)
 app.add_url_rule("/cube/study/labs", view_func=cube_study_labs_button)
+app.add_url_rule(
+    "/cube/get-microcerts", view_func=get_microcerts, methods=["GET"]
```

Comment:
Shouldn't endpoint that return json end with a .json extension? And get is implied, right? What about: `/cube/microcerts.json`?

Reply:
Done